processing/src/main/java/io/druid/query/aggregation/cardinality/types/DoubleCardinalityAggregatorColumnSelectorStrategy.java

```diff
@@ -31,12 +32,16 @@
   @Override
   public void hashRow(BaseDoubleColumnValueSelector dimSelector, Hasher hasher)
```

Comment:
"dimSelector" traditionally means `DimensionSelector`, so to avoid confusion I suggest to either call this parameter "columnSelector" or just "selector". In `FloatCardinalityAggregatorColumnSelectorStrategy`, it is called just "selector".

Reply:
done.